app/views/group_offers/_form.html.slim

```diff
@@ -1,6 +1,8 @@
 = simple_form_for(@group_offer) do |f|
   = simple_error_notice f
-
+  = f.hidden_field :user_id
```

Comment:
It would be better to assign the user in `#create` and not rely on (possibly unsafe) values sent by the client.